guide/contribute/formatting.md

```diff
@@ -85,7 +85,7 @@ There should be whitespace between paragraphs.
 ```markdown
 > The details are not the details. They make the design.
 >
-> <cite>By <a href="#">Charles Eames</a></cite>
+> This is a blockquote
```

Comment:
Did you change this intentionally? Doesn't match the code right below anymore and is also not the proper formatting for the author name.

Reply:
Ah yes. We have another rake test error with an `href="#"` on that page. How about just changing the '#' to the https://bitcoin.design URL? That should get rid of the error and users should see the proper markup.

Reply:
Fixed :)